addons/mass_mailing/static/src/builder/options/social_media_links.js

```diff
@@ -0,0 +1,219 @@
+                        platform,
+                        { href: this.baseMedias[platform], active: false },
+                    ]);
+                }
+            }
+            return {
+                medias: orderedMediaEntries.concat(inactiveMediaEntries),
+                companyId: editingElement.dataset.companyId,
+            };
+        });
+        useSortable({
+            ref: this.rootRef,
+            elements: ".hb-row",
+            handle: ".o_drag_handle",
+            cursor: "grabbing",
+            placeholderClasses: ["d-table-row"],
+            onDrop: ({ next, element }) => {
+                let nextPlatform = next?.querySelector("[data-sortable-platform]")?.dataset
+                    .sortablePlatform;
+                const elementPlatform = element.querySelector("[data-sortable-platform]").dataset
+                    .sortablePlatform;
+                const index = this.getIndex(elementPlatform, false);
+                const [media] = this.domState.medias.splice(index, 1);
+                let nextIndex = this.getIndex(nextPlatform, false);
+                this.domState.medias.splice(nextIndex, 0, media);
+                if (!media[1].active && !media[1].custom) {
+                    return;
+                }
+
+                if (!this.domState.medias.at(++nextIndex)?.[1].active) {
+                    do {
+                        ++nextIndex;
+                    } while (
+                        this.domState.medias[nextIndex] &&
+                        !this.domState.medias[nextIndex]?.[1].active
+                    );
+                    nextPlatform = this.domState.medias.at(nextIndex)?.[0];
+                }
+
+                this.dependencies.massMailingSocialMediaOptionPlugin.reorderSocialMediaLinks({
+                    editingElement: this.env.getEditingElement(),
+                    platform: elementPlatform,
+                    platformAfter: nextPlatform,
+                });
+            },
+        });
+    }
+
+    updateCompany() {
+        const fields = this.baseMedias;
+        const mappedFields = {};
+        for (const field in fields) {
+            mappedFields[`social_${field}`] = fields[field];
+        }
+        this.overlayButtonsPlugin.hideOverlayButtonsUi();
+        this.services.dialog.add(
+            ResCompanyUpdateDialog,
+            {
+                resId: parseInt(this.domState.companyId),
+                onRecordSaved: (state) => {
+                    const editingElement = this.env.getEditingElement();
+                    const platformLinks = editingElement.querySelectorAll(
+                        "[data-platform]:not([data-platform*=customLink-])"
+                    );
+                    const medias = this.medias;
+                    platformLinks.forEach((link) => {
+                        let platform = link.dataset.platform;
+                        if (platform.startsWith("customLink-")) {
```

Comment:
`platformLinks` match `:not([data-platform*=customLink-])`, why would it start with `customLink-` ?